code/modules/projectiles/guns/projectile/revolver/wayfarer.dm

```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
+	The barrel inside it refined to be ideal for 10x24 caseless and easy to clean. \
+	Well produced by Excelsior it dosnt have the normal markings other then the quote - \
+	\"The hopeless don't revolt, because revolution is an act of hope.\""
```

Comment:
"Revolution is an act of hope, thus it falls upon those willing to take up arms, to save those who will not."

Reply:
This is a real quote

Reply:
was a suggestion, closing anyway